Source/BoundaryConditions/PML.cpp

```diff
@@ -506,14 +506,26 @@ PML::PML (const BoxArray& grid_ba, const DistributionMapping& /*grid_dm*/,
         WarpX::GetInstance().getBfield_fp(0,1).ixType().toIntVect() ), dm, 2, ngb );
     pml_B_fp[2] = std::make_unique<MultiFab>(amrex::convert( ba,
         WarpX::GetInstance().getBfield_fp(0,2).ixType().toIntVect() ), dm, 2, ngb );
-
+#ifdef WARPX_MAG_LLG
+    pml_H_fp[0] = std::make_unique<MultiFab>(amrex::convert( ba,
+        WarpX::GetInstance().getHfield_fp(0,0).ixType().toIntVect() ), dm, 2, ngb );
+    pml_H_fp[1] = std::make_unique<MultiFab>(amrex::convert( ba,
+        WarpX::GetInstance().getHfield_fp(0,1).ixType().toIntVect() ), dm, 2, ngb );
+    pml_H_fp[2] = std::make_unique<MultiFab>(amrex::convert( ba,
+        WarpX::GetInstance().getHfield_fp(0,2).ixType().toIntVect() ), dm, 2, ngb );
```

Comment:
Something similar can be done for coarse patch H multifabs as well in the condiitonal` if(cgeom)`

Reply:
fixed with 482ec751